packages/components/psammead-section-label/src/index.jsx

```diff
@@ -48,9 +48,9 @@ const SectionLabelWrapper = styled.div`
   }
 
   ${({ visuallyHidden }) =>
+    // Hide for all breakpoints
```

Comment:
Can we have comments in css / styled block?

Reply:
I'll bin